cmd/tendermint/commands/run_node.go

```diff
@@ -65,7 +65,7 @@ func AddNodeFlags(cmd *cobra.Command) {
 		"proxy-app",
 		config.ProxyApp,
 		"proxy app address, or one of: 'kvstore',"+
-			" 'persistent_kvstore' or 'noop' for local testing.")
+			" 'persistent_kvstore', 'test' or 'noop' for local testing.")
```

Comment:
Should we maybe call this `'e2e'`, since that's how it's described in the README?

Reply:
Yes, that would also make sense. My reasoning for calling it 'test' was because I also have an eye on https://github.com/tendermint/tendermint/issues/5790 - we might want to consider removing some of the implementations we have of `Application` in which case we just have a single test app that someone can run with